.circleci/config.yml

```diff
@@ -98,12 +98,16 @@ workflows:
             parameters:
               python-version: *python-versions
               package: [<< pipeline.parameters.package >>]
-              constraints: ["dimod~=0.9.0,>=0.9.2 dwave-system~=1.9.0", "dimod~=0.10.0"]
+              constraints: ["dimod~=0.9.0,>=0.9.2 dwave-system~=1.9.0", "dimod~=0.10.0", "dimod==0.11.0rc0"]
```

Comment:
Do we need to pin dwave-system here together with dimod?

Reply:
We get some complaints from pip, but all seems to work